cloudquery/sdk/transformers/openapi.py

```diff
@@ -24,11 +24,24 @@ def oapi_type_to_arrow_type(field) -> pa.DataType:
         return pa.string()
 
 
-def oapi_definition_to_columns(definition: Dict) -> List[Column]:
+def get_column_by_name(columns: List[Column], name: str) -> Column:
+    for column in columns:
+        if column.name == name:
+            return column
+    return None
+
+
+def oapi_definition_to_columns(definition: Dict, override_columns=[]) -> List[Column]:
     columns = []
     for key, value in definition["properties"].items():
         column_type = oapi_type_to_arrow_type(value)
-        columns.append(
-            Column(name=key, type=column_type, description=value.get("description"))
+        column = Column(
+            name=key, type=column_type, description=value.get("description")
         )
+        override_column = get_column_by_name(override_columns, key)
+        if override_column is not None:
```

Comment:
```suggestion
        if override_column:
```

Reply:
I think better `is not None` in python in case it evaluates to false

Reply:
false would be falsy, also it's either a `Column` or `None`? I think this is the Pythonic way.